src/Files.App/Files.App.csproj

```diff
@@ -175,6 +175,7 @@
     <PackageReference Include="CommunityToolkit.WinUI.Notifications" Version="7.1.2" />
     <PackageReference Include="CommunityToolkit.WinUI.UI.Behaviors" Version="7.1.2" />
     <PackageReference Include="CommunityToolkit.WinUI.UI.Controls" Version="7.1.2" />
+    <PackageReference Include="Tulpep.ActiveDirectoryObjectPicker" Version="3.0.11" />
```

Comment:
🤔 

I'm hesitant to add another third-party package, hard much effort is involved in doing this ourselves?

Reply:
I didn’t really look into it because it was the same one in the fulltrust project. But I’ll get back to you.

Reply:
I didn't realize we were already using this, definitely not a blocker but still something to remove if possible.

Reply:
Looks to be a bit of work involved going by the project source. I would be hesitant to just try re-create it. Maybe someone else can chime in and give their opinion?

Reply:
Fully agree to leave as is right now, eventually we should be at the point where we can implement the code ourselves but it doesn't make sense to do that right now.